waku/v2/node/key.txt

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Should be in different folder - static perhaps? Also file extension seems off.

Assuming this is for tests only, we should make it clear that this is a dummy key for tests only.